src/scripts.js

```diff
@@ -44,41 +51,43 @@ apiCalls().then(data => {
 function createRecipeCards(recipes) {
     recipeSection.innerHTML = "";
     recipes.forEach(recipe => {
-        let size = (2 - (recipe.name.length / 65)).toFixed(2);
+        let size = (1.6 - (recipe.name.length / 85)).toFixed(2);
```

Comment:
I feel like we should explore a better way to do this in part 2... surely this isn't the best/only way to do this?